packages/styles/scss/components/feature-section/_feature-section.scss

```diff
@@ -44,6 +60,10 @@
     background-color: $layer-01;
     display: block;
 
+    div[class^='#{$prefix}--col'] {
+      box-sizing: border-box;
+    }
+
```

Comment:
Maybe it's about style specificity, but I'm curious as to why it was done like this over `.#{$prefix}--col`?

Reply:
i am targetting this line: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/b5ecd5f42dd85c241909d80a361cd238c58cdb94/packages/web-components/src/components/feature-section/feature-section.ts#L70 , so just thought I would target any classes starting with `.#{$prefix}--col`